### DISCUSSION TITLE
Is there any way to make a custom 404 page?

### DISCUSSION THREAD

**Discussion by blacbrue** [community/community#22764] [Other Feature Feedback, Questions, & Ideas]
<p>The title says it all</p>

**Comment by Simran-B** [comment, marked as answer]
<p>It’s all explained in the documentation:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site" target="_blank" rel="noopener nofollow ugc">Creating a custom 404 page for your GitHub Pages site - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by blacbrue** [comment]
<p>Thanks! I will take a look at it</p>
